marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/transforms/mlcp-flow-transform.sjs

```diff
@@ -76,6 +76,7 @@ function transform(content, context = {}) {
 
     if (runStepsInMemory(theFlow, options)) {
       // Let errors propagate to MLCP
+      options.throwStepError = true;
```

Comment:
This may go away once flowRunner.sjs supports job/batch documents, and thus this "if" won't be needed anymore.